acct-group/radicale/radicale-0.ebuild

```diff
@@ -0,0 +1,9 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Group for the www-apps/radicale"
+ACCT_GROUP_ID=350
```

Comment:
That won't work, as UID and GID 350 is already taken by user/group 'colord'.
Please submit a PR for reserving a free UID/GID at https://github.com/gentoo/api-gentoo-org.

Reply:
Fixed by a separate PR for gentoo-api. New numbers are 328.